{"uid":"5f58c6f8f6252835","name":"test_delete_file_to_trash","fullName":"tests.delete_to_trash.test_delete_to_the_trash#test_delete_file_to_trash","historyId":"ddc7c62c35a654c066c51d71e43bec8b","time":{"start":1684506755269,"stop":1684506755269,"duration":0},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/run/data/input.txt'","statusTrace":"@pytest.fixture()\n    def get_file_id():\n        file_path = os.path.join(os.getcwd(), Links.FILE_PATH)\n        headers = {\"Authorization\": f\"Bearer {Links.ACCESS_TOKEN}\"}\n        params = {\"name\": Links.FILE_NAME, \"parents\": [Links.PARENTS]}\n        files = {\"data\": (\"metadata\", json.dumps(params),\n                          \"application/json; charset=UTF-8\"),\n>                \"file\": open(file_path, \"rb\")}\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/run/data/input.txt'\n\nconftest.py:37: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":3,"retriesStatusChange":true,"beforeStages":[{"name":"base_url","time":{"start":1684506751627,"stop":1684506751628,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"pytestconfig","time":{"start":1684506751628,"stop":1684506751628,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"delete_output_dir","time":{"start":1684506751628,"stop":1684506751628,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"_verify_url","time":{"start":1684506751628,"stop":1684506751628,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"update_refresh_token","time":{"start":1684506755270,"stop":1684506755471,"duration":201},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"get_file_id","time":{"start":1684506755471,"stop":1684506755472,"duration":1},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/run/data/input.txt'\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.3/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.3/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.3/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 909, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/APIs-tests-coursework/APIs-tests-coursework/conftest.py\", line 37, in get_file_id\n    \"file\": open(file_path, \"rb\")}\n            ^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"testStage":{"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/run/data/input.txt'","statusTrace":"@pytest.fixture()\n    def get_file_id():\n        file_path = os.path.join(os.getcwd(), Links.FILE_PATH)\n        headers = {\"Authorization\": f\"Bearer {Links.ACCESS_TOKEN}\"}\n        params = {\"name\": Links.FILE_NAME, \"parents\": [Links.PARENTS]}\n        files = {\"data\": (\"metadata\", json.dumps(params),\n                          \"application/json; charset=UTF-8\"),\n>                \"file\": open(file_path, \"rb\")}\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/run/data/input.txt'\n\nconftest.py:37: FileNotFoundError","steps":[],"attachments":[{"uid":"7bc6167e17f97b70","name":"log","source":"7bc6167e17f97b70.txt","type":"text/plain","size":151},{"uid":"e95efb5448676201","name":"stderr","source":"e95efb5448676201.txt","type":"text/plain","size":120}],"parameters":[],"stepsCount":0,"attachmentsCount":2,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"story","value":"Проверка функции удаления файла 'input.txt' в корзину"},{"name":"feature","value":"Проверка удаления файла в корзину"},{"name":"tag","value":"positive"},{"name":"parentSuite","value":"tests.delete_to_trash"},{"name":"suite","value":"test_delete_to_the_trash"},{"name":"host","value":"fv-az571-652"},{"name":"thread","value":"2333-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.delete_to_trash.test_delete_to_the_trash"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"8f032c6fe8ca955b","status":"passed","time":{"start":1684505674856,"stop":1684505676630,"duration":1774}},{"uid":"8b9b2e4d582de304","status":"passed","time":{"start":1684505642339,"stop":1684505644670,"duration":2331}},{"uid":"1fdf05fef40b4def","status":"passed","time":{"start":1683618592994,"stop":1683618595365,"duration":2371}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":4,"skipped":0,"passed":3,"unknown":0,"total":7},"items":[{"uid":"d87255861142eb55","reportUrl":"https://llizzzzzzzzii.github.io/APIs-tests-coursework/70//#testresult/d87255861142eb55","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/run/data/input.txt'","time":{"start":1684506027034,"stop":1684506027034,"duration":0}},{"uid":"39bdbf0a659dc534","reportUrl":"https://llizzzzzzzzii.github.io/APIs-tests-coursework/69//#testresult/39bdbf0a659dc534","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/run/data/input.txt'","time":{"start":1684505800967,"stop":1684505800967,"duration":0}},{"uid":"d1fbac0b82f2b7bc","reportUrl":"https://llizzzzzzzzii.github.io/APIs-tests-coursework/68//#testresult/d1fbac0b82f2b7bc","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/run/data/input.txt'","time":{"start":1684244533685,"stop":1684244533685,"duration":0}},{"uid":"a9556b57095781b0","reportUrl":"https://llizzzzzzzzii.github.io/APIs-tests-coursework/67//#testresult/a9556b57095781b0","status":"passed","time":{"start":1683618592994,"stop":1683618595365,"duration":2371}},{"uid":"b66d58770910eedf","reportUrl":"https://llizzzzzzzzii.github.io/APIs-tests-coursework/66//#testresult/b66d58770910eedf","status":"passed","time":{"start":1683618592994,"stop":1683618595365,"duration":2371}},{"uid":"7ca4518877f4a5a9","reportUrl":"https://llizzzzzzzzii.github.io/py-pytest/65//#testresult/7ca4518877f4a5a9","status":"passed","time":{"start":1683618592994,"stop":1683618595365,"duration":2371}}]},"tags":["positive"]},"source":"5f58c6f8f6252835.json","parameterValues":[]}